doc/devel/externals.md

```diff
@@ -31,10 +31,10 @@ catala implementation and compile to OCaml (removing the `external` directive):
     ```
 
 ```shell-session
-$ clerk build _build/.../Prorata_external.ml
+$ clerk build _build/.../prorata_external.ml
 ```
 
-(beware the `_build/`, and the capitalisation of the module name)
+(beware the `_build/`, it is required here)
```

Comment:
Excellent ! Can we revert to non-capitalized generated files names in `catala-examples` and `ir-catala` then?